Tools/autotest/vehicle_test_suite.py

```diff
@@ -1540,6 +1547,14 @@ def __init__(self,
         self.ubsan_abort = ubsan_abort
         self.build_opts = build_opts
         self.num_aux_imus = num_aux_imus
+        self.generate_junit = generate_junit
+        if generate_junit:
+            try:
+                spec = importlib.util.find_spec("junitparser")
+                if spec is None:
+                    raise ImportError
+            except ImportError as e:
+                raise ImportError(f"Junit export need junitparser package.\n {e} \nTry: python -m pip install junitparser")
```

Comment:
```suggestion
                raise ImportError(f"Junit export need junitparser package.\n {e} \nTry: python3 -m pip install junitparser")
```

`python` is no longer the right program name on 22.04, you need to use `python3`